src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -363,10 +403,18 @@ public static string LocalSlotName
                 functionType.ToString(),
                 ExtensionVersion,
                 IsReplay: isReplay);
+            if (this.ShouldReplayEventsBeLogged(false))
```

Comment:
This is a little misleading to read. `ShouldReplayEventsBeLogged(false)` intuitively means "don't log replay events", but that's not what it means at all. I suggest that you call it as `ShouldReplayEventsBeLogged(isReplay: false)` - basically using named parameters - to make the meaning more clear.

Reply:
This a great improvement. Thank you!